core/src/main/java/com/schibsted/spt/data/jslt/impl/BuiltinFunctions.java

```diff
@@ -422,7 +426,7 @@ public Matcher matcher(String input) {
     }
 
     private static Pattern extractor =
-      Pattern.compile("\\(\\?<([a-zA-Z][a-zA-Z0-9]*)>");
+            Pattern.compile("\\(\\?<([a-zA-Z][a-zA-Z0-9]*)>");
```

Comment:
Please drop all these whitespace-only changes. This PR should be adding the UUID function only.